### PR TITLE
Some WordPress rules

### PR DESCRIPTION
**Pull request by Decadence**
Appended some ignore rules for WordPress

### CODE REVIEW

**Comment by Decadence** [comment]
ff510be
I think these files should be ignored to prevent it from production. Otherwise everybody can see your WP version and uses it bugs or security problems. But you have to update WP on local machine and then update production from the repo. Other way is configure your server to lock down web access to this files.

1d3c565
Just ignore useless demo plugin.

**Comment by wjramos** [comment]
This is a good addition

File: WordPress.gitignore [hunk 1]
```diff
@@ -11,3 +11,8 @@ wp-content/cache/
 wp-content/upgrade/
 wp-content/uploads/
 wp-content/wp-cache-config.php
+wp-content/plugins/hello.php
+
+/readme.html
+/license.txt
```
Comment (shiftkey) [inline WordPress.gitignore]: Could excluding the license file from source control risk subverting the GPL?

>   For example, if you distribute copies of such a program, whether
gratis or for a fee, you must give the recipients all the rights that
you have.  You must make sure that they, too, receive or can get the
source code.  **And you must show them these terms so they know their
rights.**

Comment (Decadence) [inline WordPress.gitignore]: I found this thread
http://wordpress.stackexchange.com/questions/60620/is-it-breaking-the-gpl-if-you-remove-license-txt-before-uploading

**Comment by Decadence** [comment]
Strictly speaking I don't see WP version in license.txt but anyway it's useless in repo and some programmers may want to hide the fact that they use WordPress on their sites.

**Comment by wjramos** [comment]
You are free to remove license.txt (or any other files) for production

**Comment by Decadence** [comment]
So what's about merging then?

**Comment by arcresu** [comment]
Thanks!

**Pull request merged**